content/en/platform/corda/1.5/cenm/release-notes.md

```diff
@@ -14,6 +14,27 @@ title: Release notes
 
 # Corda Enterprise Network Manager release notes
 
+## Corda Enterprise Network Manager 1.5.5
+
+In CENM 1.5.5 nodes can be quarantined using the Network Map shell. Several bugs have also been fixed, introducing a more organised operational logic when two nodes are marked as 'current'. Several UI improvements have also been made.
+
+* Revoked nodes can now be quarantined on demand, and quarantine functionality has been added to the admin shell.
+  * A node is quarantined using its hash. To find the hash, use the command `view nodeInfoHashes` in the admin shell.
+  * To view quarantined nodes, use the command: `view quarantinedNodeInfos`.
+  * To purge a node from quarantine, use its hash and the command: `run purgeQuarantinedNodeInfo nodeInfoHash <hash>`.
+  * All commands can also be found by using the `help` command in the Network Map shell.
+  * Nodes with revoked certificates will be quarantined automatically.
+* CENM 1.5.5 onwards will now use Log4j's JSON log formatting for the Network Map and Signer, improving legibility and clarity of logs.
```

Comment:
CENM now uses Log4j's JSON log formatting for the Network Map and Signer, improving legibility and clarity of logs.